pkg-hacks.js

```diff
@@ -520,7 +520,9 @@ var hackers = [
     hack: function (file, contents) {
       if (isInReactNative(file)) return
 
-      var fixed = contents.replace(/process.version/g, '"' + process.version + '"')
+      var fixed = contents
+        .replace(/global\.process\.version/g, '"' + process.version + '"')
+        .replace(/process.version/g, '"' + process.version + '"')
```

Comment:
slightly more accurate (e.g. to avoid matching `process1version`). I realize this was inaccurate before :)
```suggestion
        .replace(/process\.version/g, '"' + process.version + '"')
```

Reply:
Oh yes, I was wondering about that, fixed it in the last commit!